common/resource.go

```diff
@@ -137,9 +136,8 @@ func MustCompileKeyRE(name string) *regexp.Regexp {
 func makeEmptyBlockSuppressFunc(name string) func(k, old, new string, d *schema.ResourceData) bool {
 	re := MustCompileKeyRE(name)
 	return func(k, old, new string, d *schema.ResourceData) bool {
-		log.Printf("[DEBUG] name=%s k='%v', old='%v', new='%v'", name, k, old, new)
 		if re.Match([]byte(name)) && old == "1" && new == "0" {
-			log.Printf("[DEBUG] Suppressing diff for name=%s k=%#v old=%#v new=%#v", name, k, old, new)
+			log.Printf("[DEBUG] Suppressing diff for name=%s k=%#v patform=%#v config=%#v", name, k, old, new)
```

Comment:
typo, should. be platform

Reply:
@kthejoker thanks! will be fixed along in the other prs